src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -282,10 +282,6 @@ main.help.opt.any.verbose=\
 \  -v, --verbose              Generate verbose output on standard output
 main.help.opt.create=\
 \ Operation modifiers valid only in create mode:\n
-main.help.opt.create.normalize=\
-\  -n, --normalize            Normalize information in the new jar archive\n\
-\                             after creation. This option is deprecated, and is\n\
-\                             planned for removal in a future JDK release
```

Comment:
It does appear to be leftover from when the pack200 and the -n option were removed.

We don't usually touch the translated resources. Oracle updates the translations near the end of each release for the languages supported by Oracle JDK but there are a few translations that are probably out of date now. For removal then it's probably okay.

Reply:
> We don't usually touch the translated resources. [...] For removal then it's probably okay.

What's your recommendation here? Happy to revert changes to translated files, keeping the delete in `jar.properties`, if so desirable.

This would rely on Oracle's translation update process catching the remval and update it, ideally in all translations, not only those supported by Oracle JDK, right?

Reply:
> What's your recommendation here? Happy to revert changes to translated files, keeping the delete in `jar.properties`, if so desirable.
> 
> This would rely on Oracle's translation update process catching the remval and update it, ideally in all translations, not only those supported by Oracle JDK, right?

I think what you have is okay, I'm just saying that we don't usually touch translations when making changes. 

Oracle does a refresh the de, ja, and zh_CN translations in each release so I assume the lines you removed will get removed by the next update anyway.  @justin-curtis-lu knows more about this "process". I don't know if anyone funds or maintains the other translations now, it's possible that some of these have bit rotted in recent years.

Reply:
Yes, removing them is fine here. Otherwise we would have caught them during the L10n update anyways.